docs/standard/containerized-lifecycle-architecture/index.md

```diff
@@ -7,23 +7,21 @@ ms.date: 11/23/2018
 ---
```

Comment:
change ms.date?

Reply:
Pushed another commit, although the date for the gross of the update was 11/23/2018!

Should I update all dates for the remaining articles? they all have 11/23/2018.